pkg/diff/namespace/diff.go

```diff
@@ -301,5 +302,12 @@ func isPermission(relation *core.Relation) bool {
 }
 
 func areDifferentExpressions(existing *core.UsersetRewrite, updated *core.UsersetRewrite) bool {
```

Comment:
if this changes namespace writes will not take place in `WriteSchema` API. Is this what users expect? They may expect the schema stored to be exactly the same thing as was previously written.

Reply:
This is the correct implementation and this PR fixes a bug that was otherwise causing the namespace to be written when it should not have been. We don't store the exact schema either: it is rewritten on the way back out into a string

Reply:
so you say the schema is normalized before getting written, and this makes it so that it compares the _normalized_ schema, not just the input schema?

Reply:
No, it compares the input schema but position information is not relevant for diffing